docs/agile/scrum.md

```diff
@@ -0,0 +1,48 @@
+# Scrum
+
+## Sprint name
+Instead of using sprint names like Sprint 1, Sprint 2, and so on try to name 2022 Q4 Sprint 1, 2022 Q4 Sprint 2
+
+### Why?
+- It is easier to analyze your work after generated reports for example
+- We can [sort by string format of the date](/general-rules-and-definitions.md)
```

Comment:
I saw that you have delete the fragment part.
For me it works on localhost but with `.md/` on the end of the link.
```
[sort by string format of the date](/general-rules-and-definitions.md/#sort-by-string-format-of-the-date)
```